### PR TITLE
Task/load configuration from config json

### PR DESCRIPTION
**Pull request by stevenzeiler**
Also display domain on validator history page
Eliminates the express.js app altogether, static-only.

### CODE REVIEW

**Comment by stevenzeiler** [comment]
:+1:

**Comment by wilsonianb** [comment]
It looks like validators are no longer being sorted by total validations. Should that be handled by the api?

**Comment by stevenzeiler** [comment]
It seems reasonable to sort validators by total validations in the API, what do you think?

**Comment by stevenzeiler** [comment]
The console.log has been removed.

**Comment by wilsonianb** [comment]
Yeah, that or the correlation coefficient

**Comment by stevenzeiler** [comment]
I would say sort by the correlation coefficient, which is a more relevant data point.

File: README.md [hunk 1]
```diff
@@ -11,18 +10,18 @@ bower install
...
 ## Configuration
 
-All configuration is injected with environment variables:
+All configuration is injected with `config/config.json`
 
-````
-PORT                   (default 1337)
-VALIDATOR_REGISTRY_API (default https://api.validators.ripple.com)
-````
+For the stating and production environments Circle CI will copy
```
Comment (wilsonianb) [inline README.md]: `stating` -> `staging`

**Comment by wilsonianb** [comment]
LGTM :+1:

**Pull request merged**